public/app/features/transformers/calculateHeatmap/utils.ts

```diff
@@ -1,15 +1,9 @@
-const { abs, round, pow } = Math;
+import { guessDec, roundDec } from '@grafana/data';
```

Comment:
given that it is in global scope, can we be more explicit on what "Dec" is?
```suggestion
import { guessDecimals, roundDecimals } from '@grafana/data';
```
maybe?